Dockerfile

```diff
@@ -20,24 +27,26 @@ RUN go mod download
 ADD . .
 
 ARG opts
-RUN env ${opts} make build
+RUN env ${opts} make build-static
+RUN setcap 'cap_net_bind_service=+ep' /src/bin/blocky
+
+RUN adduser -S -D -H -u 10001 -h /app -s /sbin/nologin blocky
```

Comment:
I'm not sure, but I think this will change the user id and it could break installations with CSV query logging

Reply:
That's correct if the container user id isn't mapped to an external one.
Most modern systems use >1000 so I thought I change that accordingly.
Will revert it.

Reply:
changed it back to 100